.github/workflows/mirror-certs.yaml

```diff
@@ -12,8 +12,9 @@ jobs:
       - uses: actions/checkout@v2
       - name: Download certificates
         run: |
-          wget --no-parent -nd -r -N --retry-connrefused --wait 5 --random-wait 'http://aia.pki.va.gov/PKI/AIA/VA/'
```

Comment:
-nd (hierarchy of directories)
-N timestamp (compare timestamp)
--retry-connrefused (retry...duh)
--wait 5
--random-wait (Some web sites may perform log analysis to identify retrieval programs such as Wget by looking for statistically significant similarities in the time between requests)